src/sentry/replays/usecases/query/__init__.py

```diff
@@ -286,7 +286,12 @@ def make_full_aggregation_query(
     period_start: datetime,
     period_end: datetime,
 ) -> Query:
-    """Return a query to fetch every replay in the set."""
+    """
+    Return a query to fetch every replay in the set.
```

Comment:
Remove the line break and start the summary sentence on the first line.  Next to `"""`.